apps/admin-portal/src/pages/users.tsx

```diff
@@ -16,44 +16,125 @@
  * under the License.
  */
 
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
-import { Button, Grid } from "semantic-ui-react";
-import { UserSearch, UsersList } from "../components/users";
+import { useDispatch } from "react-redux";
+import { Button, Divider, Grid } from "semantic-ui-react";
+import { getUsersList } from "../api";
+import { AddUser, UserSearch, UsersList } from "../components/users";
+import { AlertInterface, AlertLevels } from "../models";
+import { addAlert } from "../store/actions";
 
 /**
  * Users info page.
  *
  * @return {JSX.Element}
  */
-export const UsersPage = (): JSX.Element => {
+export const UsersPage: React.FunctionComponent<any> = (): JSX.Element => {
     const { t } = useTranslation();
+    const dispatch = useDispatch();
+
     const [ searchQuery, setSearchQuery ] = useState("");
+    const [ isBasicModalOpen, setModalOpen ] = useState(false);
+    const [ isRoleModalOpen, setRoleModalOpen] = useState(false);
+    const [ usersList, setUsersList ] = useState([]);
+
+    useEffect(() => {
+        getList();
+    }, []);
+
+    const getList = () => {
+        getUsersList()
+            .then((response) => {
+                handleAlerts({
+                    description: t(
+                        "views:components.users.notifications.fetchUsers.success.description"
+                    ),
+                    level: AlertLevels.SUCCESS,
+                    message: t(
+                        "views:components.users.notifications.fetchUsers.success.message"
+                    )
+                });
+                setUsersList(response.data.Resources);
+            });
+    };
+
+    /**
+     * Dispatches the alert object to the redux store.
+     * @param {AlertInterface} alert - Alert object.
+     */
+    const handleAlerts = (alert: AlertInterface) => {
+        dispatch(addAlert(alert));
+    };
 
     /**
      * Handles the `onFilter` callback action from the
-     * application search component.
+     * users search component.
      *
      * @param {string} query - Search query.
      */
     const handleApplicationFilter = (query: string): void => {
```

Comment:
Is this eApplication filter? or User filter?

Reply:
Fixed